doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1474,22 +1578,48 @@ nRF5340 Audio
 
 The issues in this section are related to the :ref:`nrf53_audio_app` application.
 
-.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
+.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0
```

Comment:
I think the issue numbers should be in rising order? (going from highest to lowest OCT number)

Reply:
Hmm, for Audio the known issues are ordered by rising OCT ticket number in e.g. the NCS 2.4.0 release tag. It makes it easier for us to know which known issue has been added last.
It is not critical, but it would be great if we could keep this structure.

Reply:
Updated. Please check.